src/sql.rs

```diff
@@ -897,12 +897,14 @@ pub async fn remove_unused_files(context: &Context) -> Result<()> {
                 }
             }
             Err(err) => {
-                warn!(
-                    context,
-                    "Housekeeping: Cannot read dir {}: {:#}.",
-                    p.display(),
-                    err
-                );
+                if !p.ends_with(BLOBS_BACKUP_NAME) {
```

Comment:
I'd make it `p == blobdir`, it's the only dir that must exists now

Reply:
It's much muchness I think, either way you're trying to predict the future.  This way it errs on too many warnings rather than not enough.  I'll leave it as is.

Reply:
But then it should be `p != blobdir.join(BLOBS_BACKUP_NAME)` apparently